src/pages/projects/ProjectResultsAccordion.tsx

```diff
@@ -19,16 +18,7 @@ export const ProjectResultsAccordion = ({ projectId }: ProjectResultsProps) => {
   const [page, setPage] = useState(1);
   const [rowsPerPage, setRowsPerPage] = useState(itemsPerRowOptions[0]);
 
-  const registrationsQueryConfig: FetchResultsParams = {
-    project: projectId,
-    from: rowsPerPage * (page - 1),
-    results: rowsPerPage,
-  };
-  const resultsQuery = useQuery({
-    queryKey: ['registrations', registrationsQueryConfig],
-    queryFn: () => fetchResults(registrationsQueryConfig),
-    meta: { errorMessage: t('feedback.error.search') },
-  });
+  const resultsQuery = useFetchRegistrationsForProject(projectId, rowsPerPage, page);
```

Comment:
Kan bruke resultat-søkehooken vi har fra før også, da det ikke ser ut som søket her skal skille seg fra andre resultatsøk på noen måte

```suggestion
  const resultsQuery = useRegistrationSearch({ project: projectId, results: rowsPerPage, page: rowsPerPage * (page - 1) });
```